src/components/PersonLink.tsx

```diff
@@ -0,0 +1,19 @@
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const getNameClass = () => {
+    return classNames({ 'has-text-danger': person.sex === 'f' });
```

Comment:
Create a enum for `'f'`

Reply:
i did. But, can you explayn, why it`s bad ?
I just wanna understand type topick better